war/src/main/js/tooltips.js

```diff
@@ -30,23 +30,28 @@ registerTooltips()
 /**
  * Registers tooltips for the page
  * If called again, destroys existing tooltips and registers them again (useful for progressive rendering)
+ * @param {HTMLElement} container - Registers the tooltips for the given container
  */
-function registerTooltips() {
+function registerTooltips(container = document) {
```

Comment:
May not work in html unit. Think we’ve had issues before

Reply:
Hoping that's what's causing the current failing tests (https://ci.jenkins.io/blue/organizations/jenkins/Core%2Fjenkins/detail/PR-6408/36/tests/) - thanks for the heads up